packages/frontend/src/components/AddButton/AddButton.tsx

```diff
@@ -69,41 +69,56 @@ const AddButton: React.FC = () => {
         onClose={() => setOpen(false)}
         aria-labelledby="simple-dialog-title"
         open={open}
+        classes={{
+          root: classes.dialog,
```

Comment:
These classes doesn't exist in `AddButton.module.css`, is something missing from the css or should we remove these classes here?

Reply:
They should have been removed. Part of an old experiment I did.